magento/magento2
# Fixed refresh create order page on backend side

### Description (*)

This PR will fix Admin create order page is when refreshed then not showing customer selection grid

### Fixed Issues (if relevant)

1. magento/magento2https://github.com/magento/magento2/issues/21603: Adminhtml - Create order - no customer selected
2. ...

### Manual testing scenarios (*)

1. Admin > Sales > Create a new order
2. Then on Customer selection grid - Refresh this page now
3. You will see Customer selection grid again

### Contribution checklist (*)
